tests/integration/api_v3/rpc/blockchainAppsMetadata.test.js

```diff
@@ -223,7 +234,7 @@ describe('get.blockchain.apps.meta', () => {
 		expect(result.meta).toMap(metaSchema);
 	});
 
-	it('invalid request param -> invalid param', async () => {
+	it('should handle invalid request param and return invalid param', async () => {
```

Comment:
not important at all, just an alternative way of writing more similar to current pattern.

```suggestion
	it('should return invalid param when called with invalid request param', async () => {
```